com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/TestUINodes.cs

```diff
@@ -524,4 +524,49 @@ internal class TestUIPropertyNode : IStandardNode
             }
         );
     }
+
+    internal class TestUpgrade_V1 : IStandardNode
+    {
+        public static string Name = "TestUpgrade";
+        public static int Version = 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Version,
+            Name,
+            "Out = In;",
+            new ParameterDescriptor("In", TYPE.Float, GraphType.Usage.In),
+            new ParameterDescriptor("Out", TYPE.Float, GraphType.Usage.Out)
+        );
+
+        public static NodeUIDescriptor NodeUIDescriptor => new(
+            Version,
+            Name + "(V1)",
```

Comment:
This is a new discovery! The UI descriptors match up with the function descriptors based on `Name`. They need to match. However, doing this thing makes total sense.... Should this node be named "TestUpgrade(V1)" ?

Reply:
Yeah, that was my intention-- primarily to make it apparent which version is which for testing.

Reply:
Okay, this is an action item for me then.